templates/shared/_credentials-contact-us-form.html

```diff
@@ -7,33 +7,96 @@ <h1>{{h1}}</h1>
+      <form action="/marketo/submit" onsubmit="getCustomFields()" method="post" id="mktoForm_{{formid}}">
         <fieldset class="u-no-margin--bottom">
           <h3>About you</h3>
           <ul class="p-list">
-            <li  class="p-list__item  ">
-              <label for="FirstName" id="LblFirstName"  >First name:</label>
+            <li  class="p-list__item">
+              <label for="FirstName" id="LblFirstName">First name:</label>
+              {% if user_info and  "fullname" in user_info %}
+              <input id="FirstName" name="firstName" maxlength="255" aria-labelledby="LblFirstName" type="text" class="is-required" required="" aria-required="true" value="{{user_info['fullname'].split(' ')[0]}}" disabled>
+              {% else %}
               <input id="FirstName" name="firstName" maxlength="255" aria-labelledby="LblFirstName" type="text" class="is-required" required="" aria-required="true" >
+              {% endif %}
             </li>
 
-            <li  class="p-list__item  ">
-              <label for="LastName" id="LblLastName"  >Last name:</label>
+            <li  class="p-list__item">
+              <label for="LastName" id="LblLastName">Last name:</label>
+              {% if user_info and "fullname" in user_info %}
+              <input id="LastName" name="lastName" maxlength="255" aria-labelledby="LblLastName InstructLastName" type="text" class="is-required" required="" aria-required="true" value="{{user_info['fullname'].split(' ')[-1]}}" disabled>
+              {% else %}
               <input id="LastName" name="lastName" maxlength="255" aria-labelledby="LblLastName InstructLastName" type="text" class="is-required" required="" aria-required="true" >
+              {% endif %}
             </li>
 
-            <li  class="p-list__item  ">
-              <label for="Email" id="LblEmail"  >Work email:</label>
-              <input id="Email" name="email" maxlength="255" aria-labelledby="LblEmail InstructEmail" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" class="is-required" aria-required="true" autocomplete="email" >
+            <li  class="p-list__item">
+              {% if user_info and "email" in user_info %}
+              <input id="Email" name="email" maxlength="255" aria-labelledby="LblEmail InstructEmail" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" class="is-required" aria-required="true" autocomplete="email" value="{{user_info['email']}}" disabled>
+              {% else %}
+              <label for="Email" id="LblEmail">Work email*:</label>
+              <input id="Email" name="email" maxlength="255" aria-labelledby="LblEmail InstructEmail" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" class="is-required" aria-required="true" autocomplete="email">
+              <p>*Please note that Credentials are optimised for Ubuntu One accounts. If you already have an Ubuntu One account, please <a href="https://login.ubuntu.com/">login</a>.
```

Comment:
maybe we can change the link to ubuntu.com/login?next=/credentials/sign-up

Reply:
Done :+1: